h2o-py/h2o/model/extensions/scoring_history.py

```diff
@@ -69,8 +71,11 @@ def scoring_history_plot(self, timestep, metric, server=False):
             plt.title("Training Scoring History")
             plt.ylim(ylim)
             plt.plot(scoring_history[timestep], scoring_history[training_metric])
+        if save_plot_path is not None:
+            plt.savefig(fname=save_plot_path, fig=fig, **savefig)    
```

Comment:
@koniecsveta it's ok to handle the basic use-case (both because we're doing it in several methods already, and because it's convenient for the user):
```python
plt.savefig(fname=save_plot_path)
```
but for all advanced usages, that's what the container is for:
```
scoring_history_plot(…).figure.savefig(path, my_custom_params)
```
therefore, I think we should drop those `**savefig` forwarded params: it's very risky because it may prevent us from adding more params to those plot functions in the future.

Reply:
we can drop it: ea118b0